clang/include/clang/Basic/arm_sme.td

```diff
@@ -321,9 +321,18 @@ let TargetGuard = "sme2" in {
 let TargetGuard = "sme2" in {
   def SVLDR_ZT : Inst<"svldr_zt", "viQ", "", MergeNone, "aarch64_sme_ldr_zt", [IsOverloadNone, IsStreamingCompatible, IsSharedZA, IsPreservesZA], [ImmCheck<0, ImmCheck0_0>]>;
   def SVSTR_ZT : Inst<"svstr_zt", "vi%", "", MergeNone, "aarch64_sme_str_zt", [IsOverloadNone, IsStreamingCompatible, IsSharedZA, IsPreservesZA], [ImmCheck<0, ImmCheck0_0>]>;
+}
 
 //
 // Zero ZT0
 //
+let TargetGuard = "sme2" in {
```

Comment:
nit: I'm not really sure why this is formatted like:

```
let TargetGuard = "sme2" in {
 ....A....
}

let TargetGuard = "sme2" in {
 ....B....
}

let TargetGuard = "sme2" in {
 ....C....
}
```

Rather than:

```
let TargetGuard = "sme2" in {
 ....A....
 ....B....
 ....C....
}
```

Reply:
I think these were added in individually which might explain it. The blocks of instructions are separated by the comments, and maybe it makes it easier to see what individual instructions need once the list of defs grow in size? I've no objections to your suggestion, but the rest of this file doesn't really follow it I think?

Reply:
I personally don't see much value in having these separate blocks, but I'm not going hold up the patch over it. We may want to do a cleanup after all builtins are implemented to remove the unnecessary target guards.